client/deposits/list/index.tsx

```diff
@@ -220,12 +206,12 @@ export const DepositsList = (): JSX.Element => {
 
 	const downloadable = !! rows.length;
 
-	const endpointExport = async ( language: string ) => {
+	const endpointExport = async () => {
 		// We destructure page and path to get the right params.
 		// eslint-disable-next-line @typescript-eslint/no-unused-vars
 		const { page, path, ...params } = getQuery();
```

Comment:
💭  May not be caused by this PR, but it looks like this may be unused code.

```suggestion
```

We have another `getQuery()` call just below, and it would be best to destructure the props from the one `getQuery()` call.

Reply:
Thanks for spotting this. I found it to be redundant and removed it here - [7f4d666](https://github.com/Automattic/woocommerce-payments/pull/10003/commits/7f4d666c1790c6832dbd7a1d5a1b5729160e5aa3)